### PR TITLE
Add `STDEV` aggregate funtion

### PR DESCRIPTION
**Pull request by zmrdltl**
resolved : https://github.com/gluesql/gluesql/issues/609
Added `STDEV` function to ast_builder and executor.

### CODE REVIEW

File: core/src/executor/aggregate/state.rs [hunk 1]
```diff
@@ -116,6 +135,17 @@ impl<'a> AggrValue {
...
+                sum_square,
+                sum,
+                count,
+            } => {
+                let sum_expr1 = sum_square.multiply(&Value::I64(count))?;
+                let sum_expr2 = sum.multiply(&sum)?;
+                let expr_sub = sum_expr1.subtract(&sum_expr2)?;
+                let cnt_square = Value::F64(count as f64).multiply(&Value::F64(count as f64))?;
+                expr_sub.divide(&cnt_square)?.sqrt()
+            }
```
Comment (devgony) [inline core/src/executor/aggregate/state.rs:148]: Looks like you can try recursive `export` to reuse Variance
```rs
Self::Stdev {                                                                                                                               [0/1387]
                    sum_square,
                    sum,
                    count,
               } => Self::export(Self::Variance {
                   sum_square,
                   sum,
                   count,
               })?
               .sqrt()
```

**Review by devgony — APPROVED** [review]
LGTM! but i left single comment to reuse previouse code

File: core/src/executor/aggregate/state.rs (hunk 1 above)
Comment (panarch) [reply core/src/executor/aggregate/state.rs:148]: @devgony That's a really good point, thanks!
Then, how about adding `variance` local function or closure inside `Self::export` and let both `Self::Variance` and `Self::Stdev` to use it?

**Review by panarch — APPROVED** [review]
Awesome... 👍 👍 
Thanks a lot!!

**Pull request merged**